### PR TITLE
API379/API-380: Add get and list family variants endpoints

### PR DESCRIPTION
**Pull request by ahocquard**

### CODE REVIEW

**Comment by TheOnlyJoey** [comment]
Is the functionality described in the documentation also valid for the PHP-API already?

File: content/akeneo-web-api.yaml [hunk 1]
```diff
@@ -3732,6 +3812,71 @@ definitions:
...
+              x-immutable: true
+            attributes:
+              type: array
+              description: Each string of the array is an attribute code bind to the variant axis
+              items:
+                type: string
+              default: []
+            axes:
+              type: array
+              description: Each string of the array is an attribute code composing the axis
```
Comment (NolwennP) [inline content/akeneo-web-api.yaml]: "is an attribute code that will be used as axis"

File: content/akeneo-web-api.yaml [hunk 2]
```diff
@@ -1564,6 +1562,88 @@ paths:
...
+          $ref: "#/responses/403Error"
+        406:
+          $ref: "#/responses/406Error"
+  /api/rest/v1/families/{family_code}/variants/{code}:
+    get:
+      summary: Get a family variant
+      operationId: "get_families__family_code__variants__code__"
+      description: This endpoint allows you to get the information about a given family variant.
+      tags:
+        - Family variants
```
Comment (NolwennP) [inline content/akeneo-web-api.yaml]: Put "Families" instead

File: content/akeneo-web-api.yaml [hunk 3]
```diff
@@ -1564,6 +1562,88 @@ paths:
         422:
           $ref: "#/responses/422Error"
 
+  /api/rest/v1/families/{family_code}/variants:
+    get:
+      summary: Get list of family variants
+      operationId: "get_families__family_code__variants"
+      description: This endpoint allows you to get a list of family variants. Family variants are paginated and sorted by code.
+      tags:
+        - Family variants
```
Comment (NolwennP) [inline content/akeneo-web-api.yaml]: Put "Families" instead

**Pull request merged**